Makefile

```diff
@@ -23,7 +23,7 @@ TEST_E2E_TIMEOUT ?= 20m
 ifeq ($(RELEASE_BUILD),true)
 	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere
 else
-	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere libvirt
+	BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud ibmcloud_powervs vsphere libvirt
```

Comment:
@bpradipt Do we need some CI tests before adding this to the default provider list?

https://github.com/confidential-containers/cloud-api-adaptor/pull/750#discussion_r1153953076

Reply:
Hmm yeah at least some offline runs will be good. 

Tagging @stevenhorsman and @wainersm if they have anything to add.

Reply:
Yeah, I'll echo that. I don't think we should mandate publicly published test results for getting this PR merged (though we probably should before declaring any 'support statement'), but I think it would be great to get the new provider created in the e2e tests.

@Amulyam24 - there is some documentation on how to add a new provider in https://github.com/confidential-containers/cloud-api-adaptor/tree/staging/test/e2e#adding-support-for-a-new-cloud-provider, and there are examples of how to run tests in the `ibmcloud` provisioner code, but please reach out to us if you hit any issues as not many people have been through this process, so it might be difficult to follow and that's on us to try and improve!

Reply:
Thanks @stevenhorsman, I'll start looking into it and work on getting the test runs. Apart from that, is there anything pending to get this PR merged?